### PR TITLE
feat(snap): Add option to selectively autostart apps

### PR DESCRIPTION
**Pull request by farshidtz**
The snap includes two apps, one for the device service and another for the rtsp server. The apps are disabled by default and can be started/enabled together by calling:
```
snap start --enable edgex-device-usb-camera
```
or selectively:
```
snap start  --enable edgex-device-usb-camera.device-usb-camera
snap start  --enable edgex-device-usb-camera.rtsp-simple-server
```

The startup can be controlled when seeding from a gadget, by setting:
```
snap set edgex-device-usb-camera autostart=true
```
However, there was no way to do this selectively per app.

This PR makes it possible to selectively control the startup when seeding from a gadget:
```
snap set edgex-device-usb-camera apps.device-usb-camera.autostart=true
snap set edgex-device-usb-camera apps.rtsp-simple-server.autostart=true
```

Signed-off-by: Farshid Tavakolizadeh <farshid.tavakolizadeh@canonical.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports
Are there any new imports or modules? If so, what are they used for and why?

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: snap/local/helper-go/go.mod [hunk 1]
```diff
@@ -2,10 +2,6 @@ module github.com/edgexfoundry/device-camera-go/hooks
 
 require github.com/canonical/edgex-snap-hooks/v2 v2.3.0
 
-require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/stretchr/testify v1.7.1 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-)
+replace github.com/canonical/edgex-snap-hooks/v2 => github.com/farshidtz/edgex-snap-hooks/v2 v2.3.3-0.20220715151512-cfe13df6aa5f
```
Comment (farshidtz) [inline snap/local/helper-go/go.mod]: Pending:
- https://github.com/canonical/edgex-snap-hooks/pull/68

**Review by lenny-goodell — DISMISSED** [review]
LGTM

File: snap/local/helper-go/go.mod [hunk 2]
```diff
@@ -1,5 +1,7 @@
-module github.com/edgexfoundry/device-usb-camera/snap/local/helper-go
+module github.com/edgexfoundry/device-camera-go/hooks
```
Comment (MonicaisHer) [inline snap/local/helper-go/go.mod]:
```suggestion
module github.com/edgexfoundry/device-usb-camera/hooks
```

Comment (farshidtz) [reply snap/local/helper-go/go.mod]: Merge resolution error. The correct name is `github.com/edgexfoundry/device-usb-camera/snap/local/helper-go`

**Review by MonicaisHer — APPROVED** [review]
@farshidtz Thank you, looks good!

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**